### PR TITLE
Assisted Balance Max Speed and Real Life Usage

### PR DESCRIPTION
**Pull request by Asher123456789**
Creates a max speed for the AssistedBalance. It now effectively balances on a real life charge station.

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
  * project.
  */
 public class Robot extends LoggedRobot {
-  public static final boolean isCompetition = true;
+  public static final boolean isCompetition = false;
```
Comment (zbrotherton2158) [inline src/main/java/frc/robot/Robot.java:32]: change to true

File: src/main/java/frc/robot/commands/AssistedBalanceCommand.java [hunk 2]
```diff
@@ -16,12 +17,14 @@ public class AssistedBalanceCommand extends CommandBase {
...
-  private PIDController pidController = new PIDController(kPBalancing, kIBalancing, kDBalancing);
+  private final double balancingDeadzoneNumber = 2.5;
+  private double pidControllerMaxSpeed = 0.1;
+  PIDController pidController = new PIDController(kPBalancing, kIBalancing, kDBalancing);
   TunableNumber tunableP = new TunableNumber("Balancing P", kPBalancing, pidController::setP);
   TunableNumber tunableI = new TunableNumber("Balancing I", kIBalancing, pidController::setI);
   TunableNumber tunableD = new TunableNumber("Balancing D", kDBalancing, pidController::setD);
-
+  //TunableNumber TunableSpeed = new TunableNumber("PID Max Speed", pidControllerMaxSpeed, (a) -> {pidControllerMaxSpeed = a});
+
```
Comment (zbrotherton2158) [inline src/main/java/frc/robot/commands/AssistedBalanceCommand.java:27]: Remove commented code

**Review by zbrotherton2158 — APPROVED** [review]

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (Asher123456789) [reply src/main/java/frc/robot/Robot.java:32]: Done

File: src/main/java/frc/robot/commands/AssistedBalanceCommand.java (hunk 2 above)
Comment (Asher123456789) [reply src/main/java/frc/robot/commands/AssistedBalanceCommand.java:27]: Done

**Review by zbrotherton2158 — APPROVED** [review]

**Pull request merged**